src/components/formik-toggle/FormikToggle.tsx

```diff
@@ -4,7 +4,9 @@
 
 import * as React from "react";
 import styled from "styled-components";
-import {loadStyleVariables} from "../../scripts/loadStyleVariables";
+import Color from '../../helpers/color';
+
+let color = new Color();
```

Comment:
const ipv let

Reply:
Moet dit dan overal aangepast worden ? Want in het voorbeeld van Button stond het ook als een let ? 😊